### PR TITLE
Create coder profile page

### LINKED ISSUE #51: Create profile page for coder
11851019_10207309747606087_1178289037_n

### PR DESCRIPTION
**Pull request by victorbalan**
Fixes #51

### CODE REVIEW

File: app/elements/profile/profile-page.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+			id="getLatestChallenges"
+			handle-as="json"
+			on-response="onLastChallengesResponse"
+			on-error="onLastChallengesError">
+		</iron-ajax>
+		<div class="vertical layout" class="fullHeight">
+			<div class="horizontal layout" class="halfHeight">
+				<paper-material id="topLeft" elevation="1" class="card">
+					<div class="horizontal layout">
+						<iron-image sizing="content" src="http://lorempixel.com/400/400"></iron-image>
```
Comment (lorenzleutgeb) [inline app/elements/profile/profile-page.html]: there's still a link to Lorem Pixel here. Can't we fall back to the Letter as we do in the list view?

**Comment by victorbalan** [comment]
ready

**Comment by victorbalan** [comment]
still in progress

File: app/styles/main.css [hunk 2]
```diff
@@ -111,3 +111,7 @@ paper-header-panel>#mainPanel{
 .fullHeight{
 	height: 100%;
 }
+
+.halfHeight{
```
Comment (lorenzleutgeb) [inline app/styles/main.css]: Could use a space.

**Comment by lorenzleutgeb** [comment]
Why did you close this?

**Comment by victorbalan** [comment]
because /profile is wrong because we pass the user key

File: app/elements/user/challenge/user-challenge-detail.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+			margin: 16px 16px 8px 16px;
+			overflow: auto;
+			background: white;
+		}
+		#challengeWrapper >> h3 {
+			font-size: 25px;
+		}
+
+		#skillPolygonWrapper{
+			height: calc(100% - 16px);
```
Comment (pbochis) [inline app/elements/user/challenge/user-challenge-detail.html]: space between id name and curly brace

File: app/elements/user/challenge/user-challenge-detail.html [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+			text-align: center;
+			margin: 16px 16px 8px 16px;
+			overflow: auto;
+			background: white;
+		}
+		#challengeWrapper >> h3 {
+			font-size: 25px;
+		}
+
+		#skillPolygonWrapper{
```
Comment (pbochis) [inline app/elements/user/challenge/user-challenge-detail.html]: space between id name and curly brace

File: app/elements/user/user-detail.html [hunk 5]
```diff
@@ -0,0 +1,160 @@
+<link rel="import" href="/bower_components/paper-material/paper-material.html">
+<link rel="import" href="/bower_components/paper-button/paper-button.html">
+<link rel="import" href="user-info.html">
+<link rel="import" href="challenge/user-challenge-detail.html">
+
+<dom-module id="user-detail">
+
+	<style>
+		:host{
```
Comment (pbochis) [inline app/elements/user/user-detail.html]: space between the selector and curly brace

File: app/elements/user/user-detail.html [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+			display: block;
+			height: 100%;
+			width: 70%;
+			margin-left: 15%;
+			margin-right: 15%;
+		}
+		paper-material{
```
Comment (pbochis) [inline app/elements/user/user-detail.html]: space between the selector and curly brace

File: app/elements/user/user-detail.html [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+			background: white;
+		}
+		#top{
```
Comment (pbochis) [inline app/elements/user/user-detail.html]: space between the selector and curly brace

File: app/elements/user/user-detail.html [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+			height: calc(40% - 24px);
+			text-align: center;
+			margin: 16px 16px 8px 16px;
+			background: white;
+			overflow: auto;
+		}
+		#bottom{
```
Comment (pbochis) [inline app/elements/user/user-detail.html]: space between the selector and curly brace

File: app/elements/user/user-detail.html [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+			height: 60%;
+		}
+		#midLeft{
```
Comment (pbochis) [inline app/elements/user/user-detail.html]: space between the selector and curly brace

File: app/elements/user/user-detail.html [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+			margin: 8px 8px 8px 16px;
+			background: white;
+			width: calc(50% - 16px);
+			padding: 16px;
+		}
+		#midRight{
```
Comment (pbochis) [inline app/elements/user/user-detail.html]: space between the selector and curly brace

File: app/elements/user/user-detail.html [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+			margin: 8px 16px 8px 8px;
+			background: white;
+			width: calc(50% - 16px);
+			padding: 16px;
+		}
+		.letter{
```
Comment (pbochis) [inline app/elements/user/user-detail.html]: space between the selector and curly brace

File: app/elements/user/user-detail.html [hunk 12]
```diff
@@ -0,0 +1,160 @@
...
+			font-size: 16vh;
+			color: #4B4F62;
+		}
+		#skillPolygonWrapper{
```
Comment (pbochis) [inline app/elements/user/user-detail.html]: space between the selector and curly brace

File: app/elements/user/user-info.html [hunk 13]
```diff
@@ -0,0 +1,60 @@
+<link rel="import" href="../skill/skill-polygon.html">
+
+<dom-module id="user-info">
+
+	<style>
+		:host{
```
Comment (pbochis) [inline app/elements/user/user-info.html]: space between the selector and curly brace

File: app/elements/user/user-info.html [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+			display: block;
+			height: 100%;
+		}
+		.letter{
```
Comment (pbochis) [inline app/elements/user/user-info.html]: space between the selector and curly brace

File: app/elements/user/user-info.html [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+			font-size: 16vh;
+			color: #4B4F62;
+		}
+		#skillPolygonWrapper{
```
Comment (pbochis) [inline app/elements/user/user-info.html]: space between the selector and curly brace

**Comment by victorbalan** [comment]
need to merge back user-info with user detail

**Comment by victorbalan** [comment]
done

File: app/elements/user/challenge/user-challenge-detail.html [hunk 16]
```diff
@@ -0,0 +1,110 @@
...
+		}
+		#challengeWrapper {
+			height: 40px;
+			padding-top: 15px;
+			text-align: center;
+			margin: 16px 16px 8px 16px;
+			overflow: auto;
+			background: white;
+		}
+		#challengeWrapper >> h3 {
```
Comment (lorenzleutgeb) [inline app/elements/user/challenge/user-challenge-detail.html]: [There is no `>>` selector in CSS.](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Getting_started/Selectors)

**Pull request merged**